scripts/seastar-addr2line

```diff
@@ -229,6 +229,8 @@ if args.test:
     for line, expected in data:
         res = parser(line.strip())
         assert res == expected, f"{line}:\nExpected {expected}\nBut got  {res}"
+    if (args.verbose):
+        print('All', len(data), 'tests passed successfully.')
```

Comment:
Nicer: `print(f'All {len(data)} tests passed successfully.')

Reply:
Changed to f-string.